apps/els_lsp/priv/code_navigation/src/rename_variable.erl

```diff
@@ -10,3 +10,23 @@ foo(_Var) ->
 
 bar(Var) ->
   Var.
+
+-spec baz(Var) -> Var
+          when Var :: atom().
+baz(Var) ->
+  Var.
+
+-record(foo, {a :: Var,
+              b :: [Var]}).
+%% BUG: `Var' in MACRO(`Var') is not considered a variable POI
```

Comment:
thanks for finding this bug, indeed els_parser currently does not expect sub-POIs in the define name, so it does not recurse into it ([els_parser:attribute_subtrees](https://github.com/erlang-ls/erlang_ls/blob/main/apps/els_lsp/src/els_parser.erl#L972))

Reply:
Should we file a ticket about this?